components/kubeflow/launcher/component.yaml

```diff
@@ -1,22 +1,22 @@
-- {name: CleanPodPolicy,            type: String,   default: Running,   description: 'Defines the policy for cleaning up pods after the TFJob completes.'}
-- {name: PS Spec,                   type: JSON,     default: '{}',      description: 'TFJob ps replicaSpecs.'}
-- {name: Worker Spec,               type: JSON,     default: '{}',      description: 'TFJob worker replicaSpecs.'}
-- {name: Chief Spec,                type: JSON,     default: '{}',      description: 'TFJob chief replicaSpecs.'}
-- {name: Evaluator Spec,            type: JSON,     default: '{}',      description: 'TFJob evaluator replicaSpecs.'}
-- {name: Tfjob Timeout Minutes,     type: Integer,  default: 1440,      description: 'Time in minutes to wait for the TFJob to complete.'}
-- {name: Delete Finished Tfjob,     type: Bool,     default: 'True' ,   description: 'Whether to delete the tfjob after it is finished.'}
+- {name: Name,                        type: String,                         description: 'TFJob name.'}
+- {name: Namespace,                   type: String,     default: kubeflow,  description: 'TFJob namespace.'}
+- {name: Version,                     type: String,     default: v1,        description: 'TFJob version.'}
+- {name: ActiveDeadlineSeconds,       type: Integer,    default: -1,        description: 'Specifies the duration (in seconds) since startTime during which the job can remain active before it is terminated. Must be a positive integer. This setting applies only to pods where restartPolicy is OnFailure or Always.'}
+- {name: BackoffLimit,                type: Integer,    default: -1,        description: 'Number of retries before marking this job as failed.'}
+- {name: ttl Seconds After Finished,  type: Integer,    default: -1,        description: 'Defines the TTL for cleaning up finished TFJobs.'}
+- {name: CleanPodPolicy,              type: String,     default: Running,   description: 'Defines the policy for cleaning up pods after the TFJob completes.'}
+- {name: PS Spec,                     type: JsonObject, default: '{}',      description: 'TFJob ps replicaSpecs.'}
+- {name: Worker Spec,                 type: JsonObject, default: '{}',      description: 'TFJob worker replicaSpecs.'}
+- {name: Chief Spec,                  type: JsonObject, default: '{}',      description: 'TFJob chief replicaSpecs.'}
+- {name: Evaluator Spec,              type: JsonObject, default: '{}',      description: 'TFJob evaluator replicaSpecs.'}
+- {name: Tfjob Timeout Minutes,       type: Integer,    default: 1440,      description: 'Time in minutes to wait for the TFJob to complete.'}
+- {name: Delete Finished Tfjob,       type: Bool,       default: 'True' ,   description: 'Whether to delete the tfjob after it is finished.'}
 implementation:
   container:
-    image: liuhougangxa/kubeflow-tfjob-launcher:latest
+    image: nikenano/launchernew:latest
```

Comment:
@Bobgy I'm sure whether it's common practise in KFP to use image in personal registry?

Reply:
Good point,  I think we should make these images official and keep on `gcr` as is done with other images that are used for Kubeflow pipelines itself.